### ISSUE TITLE
Upgrade markdown-it-anchor to v8

### ISSUE THREAD

**Issue by nhoizey** [nhoizey/nicolas-hoizey.com#754]
Missing a container: https://github.com/valeriangalliat/markdown-it-anchor/issues/100
